pkg/vm/vm.go

```diff
@@ -831,7 +831,7 @@ func (v *VM) execute(ctx *Context, op opcode.Opcode, parameter []byte) (err erro
 		// inplace
 		e := v.estack.Peek(0)
 		i := e.BigInt()
-		e.value = stackitem.Make(i.Not(i))
+		e.value = stackitem.Make(new(big.Int).Not(i))
```

Comment:
If on the other hand we assume that every stack item is unique then we don't need this.